Assets/BossRoom/Scripts/Shared/Infrastructure/DIScope.cs

```diff
@@ -0,0 +1,333 @@
+using System;
```

Comment:
Could we move those to Utilities? They seem pretty boss room indepandant and would be great to reuse in future samples

Reply:
Absolutely.